x/authz/client/cli/tx.go

```diff
@@ -16,12 +16,19 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 	"github.com/cosmos/cosmos-sdk/version"
 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
+	"github.com/cosmos/cosmos-sdk/x/authz/exported"
 	"github.com/cosmos/cosmos-sdk/x/authz/types"
+	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
+	staking "github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
 const FlagSpendLimit = "spend-limit"
 const FlagMsgType = "msg-type"
 const FlagExpiration = "expiration"
+const FlagAllowedValidators = "allowed-validators"
```

Comment:
maybe let's just use the infinitive form for the sake of simplicity and consistency with `deny-validators` 
```suggestion
const FlagAllowValidators = "allow-validators"
```